content/millennium/r4/scheduling/appointment.md

```diff
@@ -66,22 +66,26 @@ _Implementation Notes_
 
 ### Parameters
 
- Name           | Required?                                                | Type          | Description
-----------------|----------------------------------------------------------|-------------------------------------------------------------------------------------------------
- `_id`          | Yes, or one of `patient`, `practitioner`, or `location`. | [`token`]     | The logical resource id associated with the Appointment. Example: `7890`
- `date`         | Yes when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
- `patient`      | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Patient references. Example: `1234`
- `practitioner` | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Practitioner references. Example: `4567`
- `location`     | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Location references. Example: `6789`
- `status`       | No                                                       | [`token`]     | A single or comma separated list of appointment statuses. Example: `arrived`
- [`_count`]     | No                                                       | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                                                                          | Type          | Description
+-----------------------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------
+ `_id`                 | Yes, or one of `patient`, `practitioner`, or `location`.                           | [`token`]     | The logical resource id associated with the Appointment. Example: `3005759`
+ `date`                | Yes, or `-date-or-req-period` when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment start date time with offset. Example: `2019-06-07T22:22:16.270Z`
+ `-date-or-req-period` | Yes, or `date` when using `patient`, `practitioner`, or `location`.                | [`date`]      | The Appointment start instant or the Appointment requested period date time with offset. Example: `2019-06-07T22:22:16.270Z`
```

Comment:
Let's make the first part of this line match date. i.e. 'The appointment start date with offset or the Appointment requested period date time with offset'

Reply:
updated [here|https://github.com/cerner/fhir.cerner.com/pull/575/commits/e81f3335620756bd3017f934c96d201dd3f3332f#diff-af2fad7ff605afe25e51152d541f652713df8ea51a020d1eae9bc557c158d2cbR73], [Screenshot here](https://github.com/cerner/fhir.cerner.com/pull/575#issuecomment-800425819)

Reply:
nit: let's capitalize Appointment to stay consistent.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/575/commits/76edf9abc53fb2057551d9f3d0b930d6302d3bf7#diff-af2fad7ff605afe25e51152d541f652713df8ea51a020d1eae9bc557c158d2cbR73